app/assets/stylesheets/_steps.scss

```diff
@@ -163,6 +163,16 @@ $dark-grey-2: #6f777b;
 }
 
 .step-list {
+  th {
+    font-weight: normal;
```

Comment:
If we have to override font-weight here, I wonder if `th` is the most appropriate markup to use. `td` may be better.